spec/miq_ae_uri_spec.rb

```diff
@@ -16,9 +16,9 @@
   end
 
   it "escape non-ASCII Numeric characters" do
-    hash = {"test://dev?lab$~" => "OU=serverbuildingtest, OU=dev2"}
+    hash = {"test://dev?lab$" => "OU=serverbuildingtest, OU=dev2"}
     query = described_class.hash2query(hash)
```

Comment:
Can we delete this and use `Hash#to_query` instead?  Is it just for handling the AR Objects?

Reply:
It handles AR objects: https://github.com/ManageIQ/manageiq-automation_engine/blob/master/lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_uri.rb#L10

Plus returns `nil` if the hash is empty, where to_query would be a blank string.  Not sure if there is logic relying on that result.

But I could see refactoring that method to take advantage of `to_query`.

Reply:
Ok, let's refactor another time. I vote to merge, and we can refactor another time.